TestFramework/ExtensionMethods.cs

```diff
@@ -433,5 +433,32 @@ public static void WaitForElementVisible(IWebDriver driver, By by, int? timeInSe
                 count++;
             }
         }
+
+        public static IWebElement FindElementWithWait(IWebDriver driver, By findBy, int? waitPeriod = null)
```

Comment:
Looks like this method is for a specific purpose (may be sign out). Should we change the method name to reflect that

Reply:
It's used in a few places for different elements so no need.